source/features/feature-descriptions.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCleanPathname, getCurrentBranch, getRepositoryInfo, upperCaseFirst} from '../github-helpers';
+
+function toggleHandler({delegateTarget: button}: delegate.Event): void {
+	const isHidden = select('.rgh-feature-screenshot')!.classList.toggle('d-none');
+	if (isHidden) {
+		button.textContent = 'View Screenshot';
+	} else {
+		button.textContent = 'Hide Screenshot';
+	}
+}
+
+async function init(): Promise<void | false> {
+	const currentFeature = location.pathname.split('/').pop()!.replace(/.tsx|.css/, '');
+	const featureInfo = features.featuresMeta.find(feature => feature.id === currentFeature)!;
+	if (!featureInfo) {
+		return false;
+	}
+
+	const branchSelector = await elementReady('[data-hotkey="w"]')!;
+	branchSelector!.closest('.d-flex')!.after(
+		<div className="Box mb-3">
+			<div className="Box-row d-flex flex-items-center">
+				<div className="flex-auto">
+					<strong>{upperCaseFirst(featureInfo.id.replace(/-/g, ' '))}</strong>
```

Comment:
We never do this, just use the feature ID. 
```suggestion
					<strong>{featureInfo.id}</strong>
```

But since the feature name is right above, I'd just skip it
```suggestion
```

Reply:
I think it looks better with it. It shows what we are explaining.